From: Andreas Stieger <astieger@suse.com>
Date: Thu, 18 May 2017 10:31:42 +0200
References: https://build.opensuse.org/request/show/493998
Upstream: never

trust-store test added in 
https://gitlab.com/gnutls/gnutls/commit/8d740ae87fae9c1237421dd24825b78103c5da36
need ca-certificates-mozilla to run.

[  242s] FAIL: trust-store
[  242s] =================
[  242s]
[  242s] doit:64: no certificates were found in system trust store!
[  242s] FAIL trust-store (exit status: 1)

But this would create a build cycle. Skip test.

Index: gnutls-3.5.11/tests/trust-store.c
===================================================================
--- gnutls-3.5.11.orig/tests/trust-store.c	2017-04-07 07:52:07.000000000 +0200
+++ gnutls-3.5.11/tests/trust-store.c	2017-05-18 10:33:53.537598763 +0200
@@ -44,6 +44,9 @@ static void tls_log_func(int level, cons
 
 void doit(void)
 {
+	/* building without ca-certificates-mozilla, skip test */
+	exit(77);
+
 	gnutls_certificate_credentials_t x509_cred;
 	int ret;
 
